pandas/tests/indexes/test_interval.py

```diff
@@ -615,6 +616,7 @@ def test_get_item(self, closed):
                                              closed=closed)
         tm.assert_index_equal(result, expected)
 
+    # To be removed (see #16316, #16386)
```

Comment:
Why?  Could you briefly explain in the comments?

Reply:
Those questions BTW apply to all comments where you write this.

Reply:
maybe reference test_interval_new.py directly (as well as the issues numbers), so its obvious to a reader what is happening.